depfinder/cli.py

```diff
@@ -201,8 +201,9 @@ def dump_deps(deps, keys):
                      "it for python files".format(file_or_dir))
         # directories are a little easier from the purpose of the API call.
         # print the dependencies to the console and then exit
+        ignore = args.ignore or arg.ignore.split(',')
```

Comment:
`arg.ignore.split(',')` is never going to be reached. also there's a typo there (`arg`, instead of `args`)

I think i'd rather just make this be `ignore = args.ignore.split(',')` and then it's always a list. Then the downstream functions need to be updated to expect a list.

Does that sound ok?